.gitignore

```diff
@@ -0,0 +1,5 @@
+/lib/node_modules
+/lib/.sass-cache
+*.css.map
+*.css
+
```

Comment:
#Changed .gitignore
reflected collaborator (developer) push as opposed to user push